config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'main/index'
+  
+  root 'main#index'
```

Comment:
please use the correct syntax here
you only need one of these lines

Reply:
Fixed

Reply:
please refactor to
`root controller: :main, action: :index`

Reply:
Done!